devices/src/legacy/serial.rs

```diff
@@ -124,6 +124,10 @@ impl Serial {
         Self::new(id, interrupt, None)
     }
 
+    pub fn add_out(&mut self, out: Box<dyn io::Write + Send>) {
```

Comment:
Surely it should be `set_out()` add implies multiple to me.

Reply:
Do we ever need to unset the out that has been provided? I guess not since serial is not an unpluggable device.